setup.py

```diff
@@ -7,7 +7,7 @@
 
 setup(
     name = 'OpenFisca-France',
-    version = '16.1.1',
+    version = '17.0.0',
```

Comment:
Not sure this is a major version. Is there any change brought by this PR from an OpenFisca-France user point of view ?